src/classes/STG_InstallScript.cls

```diff
@@ -268,4 +270,18 @@ global without sharing class STG_InstallScript implements InstallHandler {
             upsert orgSettings;
         }
     }
+    
+    /*******************************************************************************************************
+    * @description Populate Default Contact Language Fluency custom setting when upgrading to 1.74
+    * @return void
+    */
+    private static void setDefaultContactLanguageFluency(Version previousVersion) {
+        if(previousVersion != null && previousVersion.compareTo(new Version(1,74)) < 0) {
+            Hierarchy_Settings__c orgSettings = UTIL_CustomSettingsFacade.getSettings();
+            if(orgSettings.Default_Contact_Language_Fluency__c == null || orgSettings.Default_Contact_Language_Fluency__c == '') {
+                orgSettings.Default_Contact_Language_Fluency__c = 'Fluent';
```

Comment:
@lifewithryan I'm wondering if I should be concerned about translations here? Right now, we don't have the picklist translated. But when it's translated, then it would likely need to be a different default value here (& in the UTIL_CustomSettingsFacade class)

Maybe I should just choose the first value in the picklist as the default setting?

Reply:
@Robert Can we create a custom label for the word "Fluent" and have you grab that? That way it would be translated (I think). AS long as the translations line up we'd be good.

Reply:
@lifewithryan I don't know why I didn't think of that, but yes, a label would be great.

Reply:
Done